shaders/src/common_getters.fs

```diff
@@ -43,7 +43,14 @@ highp vec4 getResolution() {
 
 /** @public-api */
 highp vec3 getWorldCameraPosition() {
-    return frameUniforms.cameraPosition;
```

Comment:
Thinking one step further, even though `getWorldCameraPosition()` is only ever used to compute (unnormalized) view vectors in built-in shaders, I would keep it as part of the public shader API.

Of course, there's a contrast that ortho projections are invariant to forward/backward movements, yet any effects that may be built on `getWorldCameraPosition()` are not. This could be resolved by snapping the camera position to the near plane and sending that to shaders, or we can keep it as the responsibility of the client if they indeed use effects mentioned above. (I think we're fine with the latter for our needs).

Reply:
https://github.com/shapr3d/filament/pull/21#discussion_r785707426